### ISSUE TITLE
What should happen when copying content with a CANVAS tag in?`

### ISSUE THREAD

**Issue by hallvors** [w3c/clipboard-apis#8]
As of right now, the state of the canvas is lost. Should we try to do something to handle it?

See also https://code.google.com/p/chromium/issues/detail?id=459822

**Comment by garykac** [comment]
In that bug, the canvas is lost because of what appears to be a Chrome bug based on the size of the data URL generated by the canvas.

Other than generally specifying how images are copied and pasted, this doesn't seem like a spec bug.

**Comment by annevk** [comment]
If that's not defined, isn't this a bug? Especially since `<canvas>` is not like `<img>`.

**Comment by garykac** [comment]
Right: "other than generally specifying how images are copied and pasted...".
But there is nothing to do WRT the state of the canvas being lost as mentioned in that Chrome bug.

This bug is tracking adding a section to cover issues with clipboard + images/canvas.
